pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/model/ComputeFieldType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.pulsar.functions.transforms.model;
+
+public enum ComputeFieldType {
+  STRING,
+  INT32,
+  INT64,
+  FLOAT,
+  DOUBLE,
+  BOOLEAN
```

Comment:
can we create an issue to support date/times ?

Reply:
https://github.com/datastax/pulsar-transformations/issues/20